src/internet_identity/src/http/metrics.rs

```diff
@@ -160,19 +165,17 @@ fn pd_aggregation_metrics(
     mut metrics_builder: LabeledMetricsBuilder<Vec<u8>>,
 ) -> Result<(), std::io::Error> {
     for window in &[Day, Month] {
-        for domain in &[None, Some(Ic0App), Some(InternetComputerOrg)] {
```

Comment:
Is there a specific reason to remove the metrics from the non-official domain?

Reply:
Yes, because of the non-negligible storage cost and because the data is not that interesting.

Reply:
Thanks for the explanation!